__tests__/app.test.js

```diff
@@ -61,6 +61,74 @@ describe("GET api/reviews", () => {
   });
 });
 
+describe("GET api/reviews?queries", () => {
+  test("200: should respond with an array of objects, filtered by category value when passed a category query parameter", () => {
+    return request(app)
+    .get("/api/reviews?category=social_deduction")
+    .expect(200)
+    .then(({body: {reviews}}) => {
+      expect(reviews.length).toBe(11)
+      reviews.forEach((review) => {
+        expect(review.category).toBe("social deduction")
+      })
+    })
+  })
+  test("200: should respond with an array of objects, sorted by the column defined by the sort_by parameter", () => {
+    return request(app)
+    .get("/api/reviews?sort_by=votes")
+    .expect(200)
+    .then(({body: {reviews}}) => {
+      expect(reviews.length).toBe(13)
+      expect(reviews).toBeSortedBy("votes", {descending: true})
+    })
+  })
+  test("200: should respond with an array of objects, sorted by date, ascending if passed ASC as order parameter", () => {
+    return request(app)
+    .get("/api/reviews?order=ASC")
+    .expect(200)
+    .then(({body: {reviews}}) => {
+      expect(reviews.length).toBe(13)
+      expect(reviews).toBeSortedBy("created_at", {descending: false})
+    })
+  })
+  test("200: should respond appropriately when passed a combination of query parameters", () => {
+    return request(app)
+    .get("/api/reviews?category=social_deduction&sort_by=votes&order=ASC")
+    .expect(200)
+    .then(({body: {reviews}}) => {
+      expect(reviews.length).toBe(11)
+      expect(reviews).toBeSortedBy("votes", {descending: false})
+      reviews.forEach((review) => {
+        expect(review.category).toBe("social deduction")
+      })
+    })
+  })
```

Comment:
You should additionally check for a query with an existent category but which has no review yet. This should give us a 200 along with an empty array.